src/assets/notionCalSchema.ts

```diff
@@ -343,6 +349,12 @@ export const notionCalSchema = {
     'type': 'rich_text',
     'rich_text': {},
   },
+  'Property': {
+    'id': 'C%7DSX',
+    'name': 'Property',
+    'type': 'rich_text',
+    'rich_text': {},
+  },
```

Comment:
I assume this is to pass the Notion sync, but this property doesn't seem to need to exist? You should make sure with the Logistics Team.